README.md

```diff
@@ -0,0 +1,20 @@
+# Mtbseq-nf
+
+OBS: This still a Work in progress!
+
+Project with aim to create a nextflow wrapper for mtbseq workflow.
+
+Heavily based on [ngs-fzb/MTBseq_source](https://github.com/ngs-fzb/MTBseq_source)
+
+Contributions are warmly aceppted!
+
+We would like to Thank the developers of MTBseq:
+
+* Thomas A. Kohl (source code, core logic)
+* Robin Koch (source code, package building)
+* Christian Utpatel (source code, core logic)
+* Maria R. De Filippo (source code, beta test)
+* Viola Schleusener (performance comparison, beta test)
+* Patrick Beckert (annotation data, beta test)
+* Daniela M. Cirillo (Head)
+* Stefan Niemann (Head)
```

Comment:
I think instead of mentioning them in this page here, it's better to add the link of the base repo.

Reason, is there's an authorship conflict between the original authors themselves, they can resolve and update on their own repol.

Reply:
yes sure! I've added the base repo as well